### ISSUE TITLE
Remove "difficulty" and add subregions

### ISSUE THREAD

**Issue by GrimPixel** [vulnerabbity/Geoperion#13]
What does "difficulty" actually mean? I guessed its "easy" means developed countries, "hard" means third world. It turned out not to be the case. I would like to have [subregions](https://en.wikipedia.org/wiki/Subregion).

**Comment by Cwpute** [comment]
Yeah, i'd prefer the difficulty setting to be something like: easy, choosing between 2 choices; medium, choosing between 4 choices; hard, searching in the whole database of country names the one you think is the correct one.
What do you think ?
